### PR TITLE
Creating sample data 

### PR DESCRIPTION
**Pull request by marywebby**

### CODE REVIEW

**Comment by marywebby** [comment]
Hey Carlos and Alex! I have created my sample data rake task and wanted to ask for y'alls approval on if you stop any bugs before i move on, thank you!

File: lib/tasks/dev.rake [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+  product_ids = Product.pluck(:id)
+
+  users.each do |user|
+    rand(1..3).times do
+      routine = user.routines.create(name: sample_routine_names.sample)
+
+      rand(1..5).times do
+        product_id = product_ids.sample
+
+        routine.routine_products.create(product_id: product_id)
```
Comment (JoseMDO) [inline lib/tasks/dev.rake:25]: This is great logic and works wonderfully!

File: app/models/routine.rb [hunk 2]
```diff
@@ -19,4 +19,18 @@
 class Routine < ApplicationRecord
   belongs_to :user
   has_many :routine_products
+
+  validate :no_duplicate_products
+
+  private
+
+  def no_duplicate_products
```
Comment (JoseMDO) [inline app/models/routine.rb:27]: Nice addition to not have duplicate products!

**Review by JoseMDO — APPROVED** [review]
Great Job!

**Pull request merged**